src/Main.bas

```diff
@@ -705,7 +711,7 @@ Function Compile(Parameter As String = "") As Integer
 				End If
 				Split sOutput, Chr(10), res()
 				For i As Integer = 0 To UBound(res) 'Copyright
-					ShowMessages(*res(i), False)
+					'ShowMessages(*res(i), False)
```

Comment:
You are removing the compilation messages. This is the Output window, there should be a complete original, easy-to-copy text of the compilation message. For showing to anyone. Therefore, I do not agree with this change.

Reply:
OK.  Move this code "ShowMessages(*res(i), False) " before  “bFlagErr = SplitError(*res(i), ErrFileName, ErrTitle, iLine)”  at line 733